test/fixtures/shared/outputs.tf

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
I believe that we're adding descriptions to outputs and variables for tests and examples as well; we can just copy/paste the values from the root module. /cc @aaron-lane can you confirm this?